.buildconfig/ci-linux.yml

```diff
@@ -7,7 +7,7 @@ channels:
 dependencies:
   - cmake=3.23.2
   - comm=0.1.3  # earlier versions break tests with ipywidgets with latest ipykernel
-  - conan=1.59.0
+  - conan=1.64.1
```

Comment:
Was this the plan? I seem to remember we talked about moving to `>=2.0`?
Does bumping to `1.64.1` solve an issue that we're currently having?

Reply:
We can get python 3.12 on conda https://github.com/scipp/scipp/issues/3322